packages/context/src/api/paramsSerializer.ts

```diff
@@ -1,11 +1,24 @@
 import { ParamsSerializerOptions } from 'axios';
+import _ from 'lodash';
 
 export function paramsSerializer(params: Record<string, unknown>, options?: ParamsSerializerOptions) {
     const searchParams = new URLSearchParams();
 
     for (const [key, value] of Object.entries(params)) {
+        if (!validateParamKey(key) || !validateParamValue(value)) {
+            continue;
+        }
+
         searchParams.append(key, typeof value !== 'string' ? JSON.stringify(value) : value);
     }
 
     return searchParams.toString();
 }
+
+function validateParamKey(item?: string) {
+    return !_.isNil(item) && !_.isUndefined(item) && !(_.isString(item) && _.isEmpty(item));
```

Comment:
Keys are already strings.  Not sure how much value there is to validating the key.  The only thing that's arguably invalid is `''`, and even then not sure it's actually invalid, though I can see there's little reason to support an empty key so I think it's fine if we ignore it.